cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SpatialTest.scala

```diff
@@ -184,6 +184,13 @@ class SpatialTest extends DocumentingTest {
           })) {
           resultTable()
         }
+        p(
+          """
+            |For the geographic values, there is to note that the `latitude` value should always lie in the interval `[-90, 90]` and any other value
```

Comment:
The phrase `there is to note` is not correct English. I suggest changing it to `it is important to note`.